contracts/StablecoinConverter.sol

```diff
@@ -236,6 +239,8 @@ contract StablecoinConverter is EpochTokenLocker {
         uint128[] memory prices,
         uint16[] memory tokenIdsForPrice
     ) public returns (uint256) {
+        verifyAmountThreshold(prices);
+        verifyAmountThreshold(volumes);
```

Comment:
Sorry for these minimal comments, but I would put them these verifyAmount functions() after the check of the claimed amount

Reply:
Very good idea! We would lose our gas savings if we don't move them to after!

Also, I was also just in the process of the following change here as well:

```
        require(verifyAmountThreshold(prices), "At least one price lower than AMOUNT_MINIMUM");
        require(verifyAmountThreshold(volumes), "At least one executed buy amount lower than AMOUNT_MINIMUM");
```